.travis.yml

```diff
@@ -13,25 +13,32 @@
 # limitations under the License.
 
 script:
-  - BALLERINA_VERSION=${VERSION:-1.0.0}
-  - echo -e "\033[36m${BALLERINA_VERSION}\033[m"
+  # - BALLERINA_VERSION=${VERSION:-1.0.0}
+  # - echo -e "\033[36m${BALLERINA_VERSION}\033[m"
 
-  - DISTRIBUTION_NAME=ballerina-${BALLERINA_VERSION}
-  - echo -e "\033[36m${DISTRIBUTION_NAME}\033[m"
+  # - DISTRIBUTION_NAME=ballerina-${BALLERINA_VERSION}
+  # - echo -e "\033[36m${DISTRIBUTION_NAME}\033[m"
 
-  - DOWNLOAD_CHANNEL=${CHANNEL:-nightly}
-  - echo -e "\033[36m${DOWNLOAD_CHANNEL}\033[m"
+  # - DOWNLOAD_CHANNEL=${CHANNEL:-nightly}
```

Comment:
Why have we commented this out?

Reply:
This commit is just a test to fix Trvis CI. Will fix the format and comments in next PR.